### PR TITLE
adding new prpl overview for io

### PR DESCRIPTION
**Pull request by Meggin**
Hey, Addy, Dave, Gray, and co

I've created the pull request for the new PRPL doc.

I have to be honest-- there's a good bit I'd change in this doc, if there was more time.

For starters-- it feels strange having an overview and a get started that have cross-overs.

Also, one of the images doesn't have context set, and there are other stylistic approaches I'd have done a bit differently.

Dave-- you might be able to do a copy-edit review too, to make it that bit better?

I know it's IO, so I wanted to get at least something ready to roll today!

FYI-- Gray needs to be added to contributors, and I'll create a separate PR for that.

### CODE REVIEW

**Comment by petele** [comment]
@Meggin - can you take a peek at this and update the status?

**Comment by petele** [comment]
just checking in, what's the status of this?

**Comment by petele** [comment]
what's the status on this?

**Comment by petele** [comment]
Can we close this, or get an update on it?

**Comment by addyosmani** [comment]
@petele I'm catching up with Meggin about the next steps for this PR offline and we'll have an update shortly. 

tl;dr: needs to have conflicts resolved. Personal preference is we do a copy-edit and land then circle back with iterative improvements.

**Comment by petele** [comment]
Any progress on this one?

**Comment by petele** [comment]
Any updates here?

**Comment by petele** [comment]
Can we close this PR then?
